wagtail/embeds/templates/wagtailembeds/chooser/embed_chosen.js

```diff
@@ -1,4 +1,4 @@
 function(modal) {
-    modal.respond('embedChosen', '{{ embed_html|escapejs }}');
+    modal.respond('embedChosen', '{{ embed_html|escapejs }}', {{ embed_json|safe }});
```

Comment:
Shouldn't it be `embed_json|escapejs`?

Reply:
No, I don't think so... we're taking advantage of the fact that the json string is syntactically valid Javascript, and inserting it unescaped into the JS code so that the calling code receives it as a JS object rather than a string.